site/en/docs/cquery.md

```diff
@@ -357,6 +357,23 @@ This option generates output as a Graphviz-compatible .dot file. See `query`'s
 also supports [`--graph:node_limit`](/reference/query#graph-nodelimit) and
 [`--graph:factored`](/reference/query#graph-factored).
 
+### Files output {:#files-output}
+
+<pre>
+--output=files
+</pre>
+
+This option prints a list of the output files produced by each target matched
+by the query similar to the list printed at the end of a `bazel build`
+invocation. The output contains only the files advertised in the requested
+output groups as determined by the
+[`--output_groups`](/reference/command-line-reference#flag--output_groups) flag.
+
+Note: The output of `bazel cquery --output=files //pkg:foo` contains the output
+files of `//pkg:foo` in *all* configurations that occur in the build (also see
```

Comment:
Which pattern do you mean here? When you build `//pkg:foo` and it has a transitive dependency on itself? Or configurations over multiple builds?

Reply:
I'm not sure I fully understood the `cquery` semantics, but I think that this would already show up in the following situation:
* `//pkg:foo` is any target
* `//pkg:bar` depends on `//pkg:foo` via a transition

Then, a simple `bazel cquery //pkg:foo` would return two results and thus also two sets of files when used with `--output=files`.